manifests/validator.pp

```diff
@@ -0,0 +1,25 @@
+# == Class: grafana::validator
+#
+# Includes `` resource
+#
+# === Parameters
+# [*grafana_url*]
+# Grafana URL.
+#
+# [*grafana_api_path*]
+# API path to validate with.
+#
+# === Examples
+#
+#  include grafana::validator
+#
+class grafana::validator (
+  String[1] $grafana_url = 'http://localhost:3000',
```

Comment:
We could use https://github.com/puppetlabs/puppetlabs-stdlib/blob/master/types/httpurl.pp here?

Reply:
Updated